core/src/main/java/org/fao/geonet/api/records/attachments/StoreUtils.java

```diff
@@ -133,20 +133,8 @@ public static void replaceDataDir(ServiceContext context, String sourceUuid, Str
             List<MetadataResource> targetAddResources = exceptMetadataResource(sourceResources, targetResources);
             List<MetadataResource> targetUpdateResources = exceptMetadataResource(targetResources, targetDeleteResources);
```

Comment:
If targetUpdateResources  and targetAddResources  are no longer used they they should be removed.